jni_gen/examples/notif_plugin/example/lib/main.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:jni/jni.dart';
+
+// The hierarchy created in generated code will mirror the java package
+// structure. This is an implementation convenience and we may allow
+// more customization in future.
+import 'package:notif_plugin/com/example/notif_plugin.dart';
+
+JlObject activity = JlObject.fromRef(Jni.getInstance().getCurrentActivity());
+
+int i = 0;
```

Comment:
Remove `i`?

Reply:
It's used as notification ID, so that each notification will be distinct.